pp/api/rpc/rpc_api.go

```diff
@@ -53,7 +53,19 @@ type ParamDownloadFileInfo struct {
 	ReqId         string   `json:"reqid"`
 }
 
-// result for all upload and download messages
+// list: request file list
+type ParamReqFileList struct {
```

Comment:
since this file define all the types for the rpc, just name it as types.go instead of rpc_api.go